admin_ui/src/components/NavBar.vue

```diff
@@ -12,7 +12,8 @@
 
             <h1>
                 <router-link to="/">
-                    <font-awesome-icon icon="tools" />Piccolo Admin
+                    <font-awesome-icon icon="tools" />
+                    {{ siteName }} Admin
```

Comment:
How do you feel about removing the word 'Admin'? The default value coming back from the API would then have to be 'Piccolo Admin' instead of 'Piccolo'.

I can imagine someone just wanting 'My Business' rather than 'My Business Admin', if the name is long.

Likewise, if the website is for a non-English audience, they might want to translate the word 'Admin'. Most of the admin is hardcoded English anyway at the moment, but hopefully one day it'll be more multi-lingual.